src/Umbraco.Core/Logging/Logger.cs

```diff
@@ -110,135 +139,85 @@ private static bool IsTimeoutThreadAbortException(Exception exception)
         /// <inheritdoc/>
         public void Warn(Type reporting, string format)
         {
-            var logger = LogManager.GetLogger(reporting);
-            if (logger == null || logger.IsWarnEnabled == false) return;
-            logger.Warn(format);
+            var logger = Log.Logger;
+            logger?.Warning(format);
```

Comment:
This is incorrect as it doesn't take into account the type, I've fixed up in the latest commit